doc/getting_started/setup_fpga.md

```diff
@@ -103,62 +107,72 @@ This behavior is helpful to create reproducible builds and avoids Vivado modifyi
-cd $REPO_TOP/build/lowrisc_systems_chip_earlgrey_cw310_0.1/synth-vivado/
-vivado
+cd $REPO_TOP/build/lowrisc_systems_chip_earlgrey_${BOARD}_0.1/synth-vivado/vivado
 ```
 
 Finally, using the Tcl console, you can kick off the project setup with
-```console
-source lowrisc_systems_chip_earlgrey_cw310_0.1.tcl
+```sh
+source lowrisc_systems_chip_earlgrey_${BOARD}_0.1.tcl
 ```
 
-## Connecting the ChipWhisperer CW310 board
+## Connecting the ChipWhisperer FPGA board:
+
+### CW310 board
 
-The ChipWhisperer CW310 board supports different power options.
+The CW310 board supports different power options.
 It is recommended to power the board via the included DC power adapter.
 To this end:
 1. Set the *SW2* switch (to the right of the barrel connector) up to the *5V Regulator* option.
-1. Set the switch below the barrel connector to the right towards the *Barrel* option.
-1. Set the *Control Power* switch (bottom left corner, *SW7*) to the right.
-1. Ensure the *Tgt Power* switch (above the fan, *S1*) is set to the right towards the *Auto* option.
-1. Plug the DC power adapter into the barrel jack (*J11*) in the top left corner of the board.
-1. Use a USB-C cable to connect your PC with the *USB-C Data* connector (*J8*) in the lower left corner on the board.
+2. Set the switch below the barrel connector to the right towards the *Barrel* option.
+3. Set the *Control Power* switch (bottom left corner, *SW7*) to the right.
+4. Ensure the *Tgt Power* switch (above the fan, *S1*) is set to the right towards the *Auto* option.
+5. Plug the DC power adapter into the barrel jack (*J11*) in the top left corner of the board.
+6. Use a USB-C cable to connect your PC with the *USB-C Data* connector (*J8*) in the lower left corner on the board.
+### CW340 board
+
+The CW340 board should be powered via the included DC power adapter.
+To this end:
+1. Set the *Control Power* switch (top left corner, *SW7*) to the right.
+2. Ensure the *Tgt Power* switch (center of the board) is set to the right towards the *Auto* option.
+3. Plug the DC power adapter into the barrel jack (*J11*) in the top left corner of the board.
+4. Use a USB-C cable to connect your PC (*host*) with the *USB-C* connector (*J28*) in the lower left corner on the board.
 
+### Detecting the board
 You can now use the following to monitor output from dmesg:
-```console
+```sh
 sudo dmesg -Hw
 ```
 This should show which serial ports have been assigned, or if the board is having trouble connecting to USB.
 If `dmesg` reports a problem you can trigger a *USB_RST* with *SW5*.
-When properly connected, `dmesg` should identify the board, not show any errors, and the status light should flash.
-They should be named `'/dev/ttyACM*'`, e.g. `/dev/ttyACM1`.
+When properly connected, `dmesg` should identify the board, not show any errors, and the status light should flash (next to *SW5*).
+They should be named `'/dev/ttyACM*'` for CW310 and `/dev/ttyUSB*` + `/dev/ttyACM*` for CW340, e.g. `/dev/ttyACM1`.
```

Comment:
This is not quite correct: it depends on the jumpers for serial. If jumpers are all on SAM3x (JP1-4) then it should be `/dev/ttyACM*`. If it's the FTDI, it should be `/dev/ttyUSB*`

Reply:
Now I'm mentioning the jumpers.